appveyor.yml

```diff
@@ -5,8 +5,10 @@ build_script:
     echo "Build worker image $env:APPVEYOR_BUILD_WORKER_IMAGE"
 
     &"C:\Program Files (x86)\Microsoft Visual Studio\Installer\vswhere.exe"
-
-    if(($env:APPVEYOR_PULL_REQUEST_HEAD_REPO_NAME -split "/")[0] -eq ($env:APPVEYOR_REPO_NAME -split "/")[0]) {
+    
+    if($env:SKIP_GYP_BUILD -eq "true") {
```

Comment:
Where does this get set?

Reply:
@nornagon in the Appveyor UI.

Reply:
Ah, gotcha, somehow I missed that this was targeted to 3-0-x.